bcda/auth/client/ssas.go

```diff
@@ -309,6 +311,9 @@ func (c *SSASClient) GetToken(credentials Credentials, r http.Request) (string,
 		return "", &customErrors.InternalParsingError{Err: err, Msg: constants.RequestStructErr}
 	}
 
+	req.Header.Add(client.TransactionIDHeader, r.Context().Value(middleware.CtxTransactionKey).(string))
+	// the following is more or less a duplicate of the above, however it does not match what SSAS is expecting
```

Comment:
I can't find any use of the existing `transaction-id` header, so I'm a fan of just removing it.